### DISCUSSION TITLE
就这么多年，没人觉得UI很有问题吗，异常复杂。

### DISCUSSION THREAD

**Discussion by 1123123w** [vernesong/OpenClash#3085] [General]
最近的版本还好一点，一级菜单【插件配置】【覆写配置】改名了，起码能一下子知道是什么功能了。以前真的连蒙带猜。

但是想配置规则集，真的是在做阅读理解，以及捉迷藏。surge，cfw，quantumultx 这些都非常好懂。唯独openclash，我一直只会最简单的能翻。

**Comment by evan618** [comment]
刚上手的时候确实感觉会比较蒙，熟悉点后其实感觉也还好，不过针对这个UI 确实划分的有点不清除，例如MATE 的一些特性穿插在原版内内核选项中，个人认为应该单独放一个页面显示。本人也不是什么大佬，从官方文档来看clash就是一个比较复杂且支持特性比较多的软件，所以就算UI 做清晰了，实际上如果没有足够的基础知识，理解起来其实也会比较吃力。
